packages/steiger/src/app.ts

```diff
@@ -65,7 +65,3 @@ export const linter = {
     return [runRulesFx.doneData, () => watcher.close()] as const
   },
 }
-
-export function defineConfig(config: Config) {
```

Comment:
Great updates!

I have only one question about this thing.
 
Could you please explain why you decided to remove this function? I know it just returned the same object you passed it. This would cause errors like "The requested module does not provide an export named 'defineConfig'" and could be annoying for the users. And I know that the project is in the beta stage, so they should expect some APIs to change. 

But in the future, it might happen that we need to do some pre-processing on the config. In fact, I was thinking the opposite, about adding some flag like "defined" for the config that went through `defineConfig` and considering it the only valid config. Yeah, and throw an exception if it doesn't have the flag, so people get used to wrapping their configs in `defineConfig` for the sake of some far-off new but quite possible features added to `defineConfig`.

What do you think?

Reply:
Actually, I think that I removed it by mistake, thanks for bringing that up. I wanted to beef up the type inference capabilities of this function, but gave up for the moment, and then just accidentally removed the whole thing.

> But in the future, it might happen that we need to do some pre-processing on the config.

Nah, I would really prefer the `defineConfig` function to just be for typing. That seems to be the convention with other tools in the JS ecosystem, no one expects this function to be a requirement or to modify the config in any way. Besides, any processing we could do through this function could also be done in Steiger itself, without putting any burden on the user

I'll bring this function back